doc/python_reference.rst

```diff
@@ -238,6 +238,7 @@ Visualization
    plot_evoked_field
    plot_evoked_white
    plot_filter
+   plot_glass_brain
```

Comment:
how about calling this plot_stc_glass_brain and exposing an stc.plot_glass_brain method ?